addons/web/static/src/views/fields/radio/radio_field.js

```diff
@@ -72,8 +73,9 @@ export const radioField = {
     displayName: _lt("Radio"),
     supportedTypes: ["many2one", "selection"],
     isEmpty: (record, fieldName) => record.data[fieldName] === false,
-    extractProps: ({ options }) => ({
+    extractProps: ({ options, string }) => ({
         orientation: options.horizontal ? "horizontal" : "vertical",
+        string,
```

Comment:
Not sure what's the purpose of string here, but maybe a better name could be found? General remark: I would avoid string as props name